lib/AccessoryManager.js

```diff
@@ -633,4 +834,143 @@ class AccessoryManager {
   }
 }
 
+function HSVtoRGB(hue, saturation, value) {
```

Comment:
We should probably move these functions elsewhere. How about creating a `lib/utils.js` and importing them here instead?